src/components/AgreementSections/Info.js

```diff
@@ -12,107 +11,96 @@ import {
-                {agreement?.isPerpetual?.label ?? <NoValue />}
-              </div>
-            </KeyValue>
-          </Col>
+          </KeyValue>
+        </Col>
+        <Col xs={3}>
+          <KeyValue label={<FormattedMessage id="ui-agreements.agreements.renewalPriority" />}>
+            <div data-test-agreement-renewal-priority>
+              {agreement?.renewalPriority?.label ?? <NoValue />}
+            </div>
+          </KeyValue>
+        </Col>
+        {agreement?.agreementStatus?.value === statuses.CLOSED &&
           <Col xs={3}>
-            <KeyValue label={<FormattedMessage id="ui-agreements.agreements.renewalPriority" />}>
-              <div data-test-agreement-renewal-priority>
-                {agreement?.renewalPriority?.label ?? <NoValue />}
+            <KeyValue label={<FormattedMessage id="ui-agreements.agreements.reasonForClosure" />}>
+              <div data-test-agreement-reason-for-closure>
+                {agreement?.reasonForClosure?.label ?? <NoValue />}
               </div>
             </KeyValue>
           </Col>
-          {agreement?.agreementStatus?.value === statuses.CLOSED &&
-            <Col xs={3}>
-              <KeyValue label={<FormattedMessage id="ui-agreements.agreements.reasonForClosure" />}>
-                <div data-test-agreement-reason-for-closure>
-                  {agreement?.reasonForClosure?.label ?? <NoValue />}
-                </div>
-              </KeyValue>
-            </Col>
-          }
-        </Row>
-        <Row>
-          <Col xs={12}>
-            <KeyValue label={<FormattedMessage id="ui-agreements.agreements.agreementDescription" />}>
-              <div data-test-agreement-description style={{ whiteSpace: 'pre-wrap' }}>
-                {agreement.description || <NoValue />}
-              </div>
-            </KeyValue>
-          </Col>
-        </Row>
-        <Row>
-          <Col xs={12}>
-            <KeyValue label={<FormattedMessage id="ui-agreements.agreements.currentPeriodNote" />}>
-              <div data-test-agreement-current-period-note style={{ whiteSpace: 'pre-wrap' }}>
-                {agreement?.currentPeriod?.note ?? <NoValue />}
-              </div>
-            </KeyValue>
-          </Col>
-        </Row>
-        {agreement?.alternateNames?.length !== 0 &&
-          <MultiColumnList
-            columnMapping={{ name: <FormattedMessage id="ui-agreements.alternativeNames" /> }}
-            contentData={agreement.alternateNames}
-            visibleColumns={['name']}
-          />
         }
-      </div>
-    );
-  }
+      </Row>
+      <Row>
+        <Col xs={12}>
+          <KeyValue label={<FormattedMessage id="ui-agreements.agreements.agreementDescription" />}>
+            <div data-test-agreement-description style={{ whiteSpace: 'pre-wrap' }}>
+              {agreement.description || <NoValue />}
+            </div>
+          </KeyValue>
+        </Col>
+      </Row>
+      {agreement?.alternateNames?.length !== 0 &&
```

Comment:
Change made

Reply:
Although I believe that was existing code I've ported over

Reply:
Turns out this was necessary. 
```
agreement?.alternateNames?.length &&
```

displayed a 0 for the length value, rather than using it as a truthy check. Reverted to previous code